addons/emoji-picker/userstyle.css

```diff
@@ -1,4 +1,5 @@
 .sa-emoji-picker {
+  display: inline-block;
```

Comment:
`.sa-emoji-button-container` also needs this style. It currently looks like this in this PR:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/51849865/122a1666-8dd6-4205-9f56-995459826cf6)

On master:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/51849865/c41022f6-51bd-45cf-8ff4-f382fa19a36b)